### PR TITLE
sonar cloud issues fixed

### PR DESCRIPTION
**Pull request by MuthuKumari-P**

### CODE REVIEW

File: src/main/java/in/muthukumari/servlet/CustomerDetailServlet.java [hunk 1]
```diff
@@ -38,29 +43,41 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
...
+			if (isAdded) {
+				response.sendRedirect("DisplayCustomerBankDetail.jsp?name=" + name + "&bankName=" + bankName
+						+ "&branchName=" + branchName + "&ifsc=" + ifsc + "&accNumber=" + accNumber + "&atmNum="
+						+ atmNum + "&atmPinNum=" + atmPinNumber + "&mobileNum=" + mobileNumber + "&balanceAmount="
+						+ balanceAmount);
+			} else {
+				String errorMsg = "Invalid Data";
+				response.sendRedirect("CustomerBankDetail.jsp?errorMsg=" + errorMsg);
+			}
+		} catch (IOException | DBException | CustomerRepeatedException | ClassNotFoundException | SQLException e) {
```
Comment (selva08sankari) [inline src/main/java/in/muthukumari/servlet/CustomerDetailServlet.java]: handle DB and service exception in service class

File: src/main/java/in/muthukumari/servlet/GetIfscServlet.java [hunk 2]
```diff
@@ -41,7 +45,7 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
 			out.print(json);
 			out.flush();
 		} catch (ClassNotFoundException | InvalidException | SQLException e) {
```
Comment (selva08sankari) [inline src/main/java/in/muthukumari/servlet/GetIfscServlet.java]: this also

File: src/main/java/in/muthukumari/validator/BankDetailValidator.java [hunk 3]
```diff
@@ -23,18 +27,121 @@ public static boolean isValidAccountNumber(CustomerBankDetail customer) throws D
...
 		Map<String, Integer> bankAndAccountNumberLengthList = AccountNumberLength.getAccountNumberLength();
 		if (bankAndAccountNumberLengthList.containsKey(customer.getBankName())) {
 			int accountNumberLength = bankAndAccountNumberLengthList.get(customer.getBankName());
-			if (accountNumberLength == lengthOfAccountNumberStr) {
+			if (accountNumberLength == lengthOfAccountNumberStr &&  accountNumberStr.matches(regex)) {
 				isValidAccountNumber = true;
 			}
-		}
-		else {
+		} else {
```
Comment (selva08sankari) [inline src/main/java/in/muthukumari/validator/BankDetailValidator.java:38]: DB exception only throw from D.A.O not throwed from validator

**Review by selva08sankari — APPROVED** [review]

File: src/main/java/in/muthukumari/servlet/CustomerDetailServlet.java (hunk 1 above)
Comment (MuthuKumari-P) [reply src/main/java/in/muthukumari/servlet/CustomerDetailServlet.java]: I have Solved this issue

File: src/main/java/in/muthukumari/servlet/GetIfscServlet.java (hunk 2 above)
Comment (MuthuKumari-P) [reply src/main/java/in/muthukumari/servlet/GetIfscServlet.java]: I did changed this

File: src/main/java/in/muthukumari/validator/BankDetailValidator.java (hunk 3 above)
Comment (MuthuKumari-P) [reply src/main/java/in/muthukumari/validator/BankDetailValidator.java:38]: I have changed this

**Pull request merged**